lib/ez_gen/template.rb

```diff
@@ -0,0 +1,139 @@
+class Template
```

Comment:
This class may as well be a module if it has no instance variables. I believe you can keep the same syntax `class << self` with modules. Everything should continue to work the same as it does now.

Reply:
I changed this to a module `Templatable` and removed the `class << self` syntax as it was made unnecessary after converting to a module.